metricbeat/module/elasticsearch/cluster_stats/data.go

```diff
@@ -19,29 +19,138 @@ package cluster_stats
 
 import (
 	"encoding/json"
-
-	"github.com/pkg/errors"
-
+	"fmt"
 	"github.com/elastic/beats/v7/libbeat/common"
 	s "github.com/elastic/beats/v7/libbeat/common/schema"
 	c "github.com/elastic/beats/v7/libbeat/common/schema/mapstriface"
+	"github.com/elastic/beats/v7/metricbeat/helper/elastic"
 	"github.com/elastic/beats/v7/metricbeat/mb"
 	"github.com/elastic/beats/v7/metricbeat/module/elasticsearch"
+	"github.com/pkg/errors"
+	"hash/fnv"
+	"sort"
+	"strings"
 )
 
 var (
 	schema = s.Schema{
 		"status": c.Str("status"),
 		"nodes": c.Dict("nodes", s.Schema{
-			"count":  c.Int("count.total"),
-			"master": c.Int("count.master"),
-			"data":   c.Int("count.data"),
+			"count": c.Ifc("count"),
+			"ingest": s.Object{
+				"pipelines": s.Object{
+					"count": c.Int("ingest.number_of_pipelines"),
+				},
+				//TODO Not sure if to remove this one
+				"processor_stats": c.Ifc("ingest.processor_stats"),
```

Comment:
Did you decide to keep this field? If so, remove the `TODO` comment so it's not confusing in the future. If not, what information are you looking for to make the decision? Is this field being used by the UI - if not, I think we can remove it then?

Reply:
Removed!